packages/e2e-tests/tests/repaint-06.test.ts

```diff
@@ -0,0 +1,126 @@
+import { openDevToolsTab, startTest } from "../helpers";
+import { findConsoleMessage, seekToConsoleMessage } from "../helpers/console-panel";
+import { stepOver } from "../helpers/pause-information-panel";
+import { getGraphicsDataUrl, waitForRepaintGraphicsToLoad } from "../helpers/screenshot";
+import { debugPrint } from "../helpers/utils";
+import { expect, test } from "../testFixture";
+
+test.use({ exampleKey: "doc_control_flow.html" });
+
+test.skip("repaint-06: repaints the screen screen when stepping over code that modifies the DOM", async ({
+  pageWithMeta: { page, recordingId },
+  exampleKey,
+}) => {
+  await startTest(page, recordingId);
+  await openDevToolsTab(page);
+
+  const printedStrings = [
+    "catch",
+    "afterCatch",
+    "finally",
+    "yield 1",
+    "generated 1",
+    "yield 2",
+    "generated 2",
+    "after timer 1",
+    "after timer 2",
+    "after throw timer 1",
+    "after throw timer 2",
+    "within iterator",
+  ];
+
+  const paints: {
+    [key: string]: {
+      before: string | null;
+      after: string | null;
+    };
+  } = {};
+
+  {
+    await debugPrint(page, "Testing repaint graphics");
+
+    let graphicsDataUrl: string | null = null;
+    let previousGraphicsDataUrl: string | null = null;
+
+    for (let index = 0; index < printedStrings.length; index++) {
+      const string = printedStrings[index];
+      const text = `updateText("${string}")`;
+
+      const locator = await findConsoleMessage(page, text, "console-log");
+      await seekToConsoleMessage(page, locator);
+
+      // Step to "Element text before..."
+      await stepOver(page);
+      await stepOver(page);
+
+      await waitForRepaintGraphicsToLoad(page);
+
+      graphicsDataUrl = await getGraphicsDataUrl(page);
+      expect(
+        graphicsDataUrl,
+        `Expected graphics before "${string}" not to match previous after graphics`
+      ).not.toEqual(previousGraphicsDataUrl);
```

Comment:
I don't think this is correct: why should the screenshot change in between updates?

Reply:
Yeah, you're right about that.